invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -49,7 +49,7 @@ body {
   }
 }
 
-button:focus-visible, a:focus-visible {
+button:focus-visible, a:focus-visible, [tabindex="0"]:focus-visible {
```

Comment:
Honestly I don't remember exactly which element this was, as I made this change some weeks ago. But as long as an element has `tabindex: 0` i.e. it can be focused, it should have a visible focus.